<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuickActions</class>
 <widget class="QDockWidget" name="QuickActions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quick actions</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QTreeView" name="treeView">
      <property name="rootIsDecorated">
       <bool>true</bool>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionNew_quick_action">
   <property name="icon">
    <iconset resource="../../mgdt.qrc">
     <normaloff>:/mgdt/icons/yellow_note.png</normaloff>:/mgdt/icons/yellow_note.png</iconset>
   </property>
   <property name="text">
    <string>New quick action</string>
   </property>
  </action>
  <action name="actionRemove_quick_action">
   <property name="icon">
    <iconset resource="../../mgdt.qrc">
     <normaloff>:/mgdt/icons/red_minus.png</normaloff>:/mgdt/icons/red_minus.png</iconset>
   </property>
   <property name="text">
    <string>Remove quick action</string>
   </property>
  </action>
  <action name="actionExecute_quick_action">
   <property name="icon">
    <iconset resource="../../mgdt.qrc">
     <normaloff>:/mgdt/icons/blue_top_left_arrow.png</normaloff>:/mgdt/icons/blue_top_left_arrow.png</iconset>
   </property>
   <property name="text">
    <string>Execute quick action</string>
   </property>
  </action>
  <action name="actionModify_quick_action">
   <property name="icon">
    <iconset resource="../../mgdt.qrc">
     <normaloff>:/mgdt/icons/blue_tool.png</normaloff>:/mgdt/icons/blue_tool.png</iconset>
   </property>
   <property name="text">
    <string>Modify quick action</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../mgdt.qrc"/>
 </resources>
 <connections/>
</ui>
